### ISSUE TITLE
Not logged in user > My Signals > Log in form > Navigate back > User is able to see the menu

### ISSUE THREAD

**Issue by pezekieva** [HelpAPaw/Android#70]
### Description
Minor issue in case a user is not logged in and tries to open My Signals menu, then they are redirected to the Login page, but instead of logging in, they navigate back - they are redirected to the 'My Signals' menu, which is supposed to not be accessible. 

**Steps to reproduce**
1. Open the application and do not log in
2. Go to the side menu > Select 'My Signals' -> A pop-up is displayed informing the user that this page is restricted to registered users
3. On the pop-up, click on 'OK" -> User is redirected to Login page
4. Do not log in
5. Instead, use the Android navigation to go back to the previous page

**Expected result**
The user is redirected back to the previous page and is still not able to open 'My Signals' menu, as they're still not logged in.

**Actual result**
The user is redirected to 'My Signals' menu. As a user, I would consider this not ideal, although it's not a big issue.
Please see[ a video of the described behavior](https://www.screencast.com/t/wVbIh03zQD).

**Test Data**
version - 5.1.0(82)

**Comment by medeaohm** [comment]
Да, тук наистина е малко странно, но трябва да помислим как би било по-добре да се държи. Очакваме ли при натискаме на back бутона да се върнем на My Signals и отново да ни изкара pop-up-a? Аз честно казано като user не съм сигурна как бих предпочела да се държи, може би за сега не си заслужава да го променяме?

**Comment by BurningAXE** [comment]
Реално потребителят вижда празен екран, което е очаквано след като е видял съобщението и умишлено не се е логнал. Не мисля, че си заслужава да отделяме време за това.
